spec/unit/validator_spec.rb

```diff
@@ -88,7 +88,7 @@ def podspec_path(name = 'JSONKit', version = '1.4')
         validator.quick = true
         validator.stubs(:validate_url)
         validator.validate
-        validator.validation_dir.should.be == Pathname.new(Dir.tmpdir) + 'CocoaPods/Lint'
```

Comment:
I think it originated with making sure we used `realpath` but I agree, it's not great. I'm ok with removing it